inputs/TransactionForm.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import { Text, View, TextInput, TouchableOpacity, Platform } from 'react-native';
+import DateTimePicker from '@react-native-community/datetimepicker';
+import styles from '../styles';
+
+export default function TransactionForm() {
+    const [description, setDescription] = useState('');
+    const [amount, setAmount] = useState('');
+    const [date, setDate] = useState(new Date());
+    const [showDatePicker, setShowDatePicker] = useState(false);
+
+    const handleDateChange = (event: any, selectedDate: Date | undefined) => {
```

Comment:
In the input lesson, we showed how to update state directly as a prop on change rather than a separate function because "any" is not a good practice and opts out of type checking completely. Also, if something is optional, so it can be undefined, simplistic way would be to add ``?`` to it, such as ``selectedDate?: Date``